### ISSUE TITLE
Slider Thumbs can't switch focus in IE11

### ISSUE THREAD

**Issue by desig9stein** [IgniteUI/igniteui-angular#6170]
## Description  
If you have a slider with two thumbs and you click to drag one of them you are unable to select the other. This is reproducible only by using the mouse, the keyboard navigation is working just fine.

 * igniteui-angular version: all
 * browser: IE11

## Steps to reproduce  

1. Open slider dev demo inside IE11
2. Click on the thumb(**From**)
3. Click on the thumb(**To**)
Note: This is happening also if you start from the "**To**" to "**From**" thumb

## Result  
The user is unable to select the Thumb(To)

## Expected result  
The user should be able to select the thumb

## Attachments  
Attach a sample if available, and screenshots, if applicable.

**Comment by Aleksandyr** [comment]
@desig9stein This is a third-party issue. It seems that in IE11 **focus** and **blur** events are fired asynchronously which is not aligned with the [W3C standard](https://www.w3.org/TR/uievents/#focus) behavior.
This leads to the following problem:
Once you try to change the focus of the thumbs and click to the one that hasn't been focused, the update of the thumb happens before the evaluation of the listener. If you try to extract the active element once you end the described steps above, you will determine that the focus lays on the right thumb.
Unfortunately, there is no easy way to fix the problem.
